### PR TITLE
emissions schedule update

### PR DESCRIPTION
**Pull request by samgutentag**
updating emissions schedule and style guide fixes. revising sample calculations

### CODE REVIEW

File: docs/blockchain/mining/mining.mdx [hunk 1]
```diff
@@ -11,21 +11,20 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+Network. Hotspots are rewarded in Helium Token, HNT.
 
 Every epoch, the current consensus group mines approximately 30 blocks on the blockchain. In each
 block, Hotspots perform various types of work and are awarded according to the following
 distribution:
 
-
 | Reward Type           | Chain Variable          | Description                                                                                                         |
 | :-------------------- | :---------------------- | :------------------------------------------------------------------------------------------------------------------ |
 | PoC Challenger        | poc_challengers_percent | Rewarded to any Hotspot that creates a valid PoC challenge and submits the corresponding receipt to the blockchain. |
```
Comment (meowshka) [inline docs/blockchain/mining/mining.mdx]: Hotspots don't create challenges anymore, validators do.

File: docs/blockchain/mining/mining.mdx [hunk 2]
```diff
@@ -35,16 +34,15 @@ distribution:
...
 | Network Data Transfer | dc_percent              | Distributed each epoch to Hotspots that route LongFi sensor data for sensors on the Network during that epoch.      |
 
-
 **_Do I Have To Actively Participate to Earn Rewards Once My Hotspot is Deployed?_**
 
-No. Once your Hotspot is completely deployed and fully synced with the Helium blockchain, you as the
-owner are not required to do anything else in order to earn $HNT. Your Hotspot will perform all of
-the above activities on its own while it runs.
+No. Once your Hotspot is completely deployed and onboarded to the Helium blockchain, you as the
+owner are not required to do anything else to earn HNT. Your Hotspot will perform all of the above
```
Comment (meowshka) [inline docs/blockchain/mining/mining.mdx:49]: Hotspots don't create challenges.

File: docs/blockchain/mining/mining.mdx [hunk 3]
```diff
@@ -56,94 +54,93 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
-| PoC Challenger        | 0.90% - 2.11%   | 15.6250 - 36.6319         |
-| PoC Challengees       | 5.02% - 11.78%  | 87.1527 - 204.5139        |
-| Witnesses             | 20.08% - 47.11% | 348.6111 - 817.8819       |
-| Consensus Group       | 6%              | 104.1666                  |
-| Security Tokens       | 33%             | 572.9166                  |
-| Network Data Transfer | _Up to 35%_     | _Up to 607.6389_          |
-| **Total**             | **100%**        | **1736.1111**             |
+| Reward Type           |     Percentage | HNT Earned by Reward Type |
+| :-------------------- | -------------: | ------------------------: |
+| PoC Challenger        |          0.85% |                   14.7569 |
```
Comment (meowshka) [inline docs/blockchain/mining/mining.mdx]: Why were the ranges removed?

Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: this was following cokes numbers on discord, what do the ranges represent?

File: docs/blockchain/mining/mining.mdx [hunk 4]
```diff
@@ -56,94 +54,93 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
+- All Hotspots participating in PoC, including `Challengees` and `Witnesses` will earn rewards
+  proportional to how many events they participated in out of the total number of events per epoch.
 - All Hotspots participating in `Network Data Transfer` will earn rewards proportional to their
   share of the total data transfer in that epoch, as shown above.
 - Hotspots can earn one or more reward types during any given epoch.
-- Hotspots are only eligible to submit one Proof of Coverage Challenge - which results in them
-  earning the `PoC Challenger` reward - **up to once per 360 blocks**.
-- `PoC Challenger`, `PoC Challengee`, and `Witness` reward types get distributed in the epoch that
-  includes the corresponding PoC receipt.
+- `PoC Challengee` and `Witness` reward types get distributed in the epoch that includes the
```
Comment (meowshka) [inline docs/blockchain/mining/mining.mdx]: What about "PoC Challenger" reward?

File: docs/blockchain/mining/mining.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: commit inbound

File: docs/blockchain/mining/mining.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx:49]: fixed with next commit

File: docs/blockchain/mining/mining.mdx (hunk 4 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: ah, yes, was formatting this to be hotspot only, this section includes Validators, fixed no next commit

File: docs/blockchain/mining/mining.mdx (hunk 3 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: resolved in latest commit

File: docs/blockchain/mining/mining.mdx [hunk 5]
```diff
@@ -56,92 +53,91 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
+- These `10 HNT` would be split proportionally to the Hotspots that did the work routing packets at
   the 1:1 rate.
-- The remaining `597.6389` from the Network Data Transfer reward would be distributed ratably among
-  the Challengers, Witnesses and Challengees.
-  - `20.6457` to the Challenger group
-  - `461.5945` to the Witness group
-  - `115.3986` to the Challengee group
+- The remaining `641.0417` from the Network Data Transfer reward would be distributed by weight
+  among the Challengers, Witnesses, and Challengees.
+  - `22.2402` to the Challenger group => `0.85/(0.85+4.73+18.92) = 3%`
```
Comment (meowshka) [inline docs/blockchain/mining/mining.mdx]: Challenger doesn't get leftovers.

Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: fixed! thank you

File: docs/blockchain/mining/mining.mdx [hunk 6]
```diff
@@ -11,40 +11,37 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
-| Consensus Group       | consensus_percent       | Divided equally among the Validators that are part of the outgoing Consensus Group, responsible for mining blocks.  |
-| Security              | securities_percent      | Awarded to Helium, Inc and other Network investors who hold Security Tokens.                                        |
-| Network Data Transfer | dc_percent              | Distributed each epoch to Hotspots that route LongFi sensor data for sensors on the Network during that epoch.      |
-
+| Reward Type           | Chain Variable          | Description                                                                                                        |
+| :-------------------- | :---------------------- | :----------------------------------------------------------------------------------------------------------------- |
+| PoC Challengees       | poc_challengees_percent | Awarded to any Hotspot that transmits a PoC packet after being targeted by the challenger.                         |
+| Witnesses             | poc_witnesses_percent   | Distributed to all Hotspots that witness a beacon packet as part of a PoC Challenge.                               |
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx]:
```suggestion
| Witnesses             | poc_witnesses_percent   | Distributed to selected Hotspots that witness a beacon packet as part of a PoC Challenge.                               |
```

Comment (abhay) [reply docs/blockchain/mining/mining.mdx]: slightly more clear

File: docs/blockchain/mining/mining.mdx [hunk 7]
```diff
@@ -11,40 +11,37 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
-| Witnesses             | poc_witnesses_percent   | Distributed to all Hotspots that witness a beacon packet as part of a PoC Challenge.                                |
-| Consensus Group       | consensus_percent       | Divided equally among the Validators that are part of the outgoing Consensus Group, responsible for mining blocks.  |
-| Security              | securities_percent      | Awarded to Helium, Inc and other Network investors who hold Security Tokens.                                        |
-| Network Data Transfer | dc_percent              | Distributed each epoch to Hotspots that route LongFi sensor data for sensors on the Network during that epoch.      |
-
+| Reward Type           | Chain Variable          | Description                                                                                                        |
+| :-------------------- | :---------------------- | :----------------------------------------------------------------------------------------------------------------- |
+| PoC Challengees       | poc_challengees_percent | Awarded to any Hotspot that transmits a PoC packet after being targeted by the challenger.                         |
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx]: we should bring back `poc_challengers_percent` and describe where it's going.

File: docs/blockchain/mining/mining.mdx [hunk 8]
```diff
@@ -56,92 +53,90 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
-| Witnesses             | 20.08% - 47.11% | 348.6111 - 817.8819       |
-| Consensus Group       | 6%              | 104.1666                  |
-| Security Tokens       | 33%             | 572.9166                  |
-| Network Data Transfer | _Up to 35%_     | _Up to 607.6389_          |
+| PoC Challenger        | 0.85%           | 14.7569                   |
+| PoC Challengees       | 4.73% - 12.23%  | 82.1181 - 212.3264        |
+| Witnesses             | 18.92% - 48.92% | 328.4722 - 849.3056       |
+| Consensus Group       | 6%              | 104.1667                  |
+| Security Tokens       | 32%             | 555.5556                  |
+| Network Data Transfer | _Up to 37.50%_  | _Up to 651.0417_          |
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx]:
```suggestion
| PoC Challenger        | 0.85%           | 14.7236                   |
| PoC Challengees       | 4.73% - 12.23%  | 82.1247 - 212.331        |
| Witnesses             | 18.92% - 48.92% | 328.4989 - 849.3323       |
| Consensus Group       | 6%              | 104.1667                  |
| Security Tokens       | 32%             | 555.5556                  |
| Network Data Transfer | _Up to 37.50%_  | _Up to 651.0417_          |
```

Comment (abhay) [reply docs/blockchain/mining/mining.mdx]: I think this is more correct.

File: docs/blockchain/mining/mining.mdx [hunk 9]
```diff
@@ -56,92 +53,90 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
 **Rewards Change Over Time**
 
-In the above table you can see the maximum allotted rewards per reward type in accordance with HIP
-10 (described below). If you want to see the current reward type percentage, you can always query
-the [Chain Variables API](/api/blockchain/chain-variables). The next maximum allotted rewards change
-is currently scheduled for **August 1, 2022.**
+In the above table, you can see the maximum allotted rewards per reward type following HIP 10
+(described below). If you want to see the current reward type percentage, you can always query the
+[Chain Variables API](/api/blockchain/chain-variables). The next maximum allotted rewards change is
+currently scheduled for **August 1, 2023.**
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx]:
```suggestion
currently scheduled for **August 1, 2023** and will be updated along with changes related to the havening to maintain max HNT supply.
```

File: docs/blockchain/mining/mining.mdx [hunk 10]
```diff
@@ -56,92 +53,90 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
+  all Hotspots that did the work over this epoch will split the full `1000 HNT` proportionally.
 
 **Additional Notes on Reward Types and Payouts**
 
 - All Validators in the `Consensus` group will earn an equal reward.
-- All Hotspots participating in PoC, including `Challengers`, `Challengees` and `Witnesses` will
-  earn rewards proportional to how many events they participated in out of the total number of
-  events per epoch.
+- All Hotspots participating in PoC, including `Challengees` and `Witnesses` will earn rewards
+  proportional to how many events they participated in out of the total number of events per epoch.
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx:147]: suggestion: add a note about validators being challengers here

File: docs/blockchain/mining/mining.mdx [hunk 11]
```diff
@@ -56,92 +53,90 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
...
 - All Hotspots participating in `Network Data Transfer` will earn rewards proportional to their
   share of the total data transfer in that epoch, as shown above.
 - Hotspots can earn one or more reward types during any given epoch.
-- Hotspots are only eligible to submit one Proof of Coverage Challenge - which results in them
-  earning the `PoC Challenger` reward - **up to once per 360 blocks**.
```
Comment (abhay) [inline docs/blockchain/mining/mining.mdx:144]: add a description of the new targeting / poc keys structure introduced in HIP 55

File: docs/blockchain/mining/mining.mdx (hunk 7 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx]: added, with an explainer note callout to match

File: docs/blockchain/mining/mining.mdx (hunk 11 above)
Comment (samgutentag) [reply docs/blockchain/mining/mining.mdx:144]: Added a quick blurb on Validators using `poc_challenge_rate` and a link to the content in HIP55; I still need to get my head around this before I add any additional information.  Flagged on my end to continue

**Review by meowshka — APPROVED** [review]

**Pull request merged**